src/scanners/javascript.js

```diff
@@ -48,10 +48,15 @@ export default class JavaScriptScanner {
         parserOptions: {
           ecmaVersion: 2017,
         },
-        ignore: false,
         rules: _ruleMapping,
         plugins: ['no-unsafe-innerhtml'],
         allowInlineConfig: false,
+
+        // Disable ignore-mode but overwrite eslint default ignore patterns
+        // since these are matched nonetheless. Fixes #1288
```

Comment:
Can you link to the actual issue URL here instead of the number? It will make it easier for newbies, etc. to get to the actual issue.